reuseport/reuseport_windows.go

```diff
@@ -0,0 +1,27 @@
+// Package reuseport provides TCP net.Listener with SO_REUSEPORT support.
+//
+// SO_REUSEPORT allows linear scaling server performance on multi-CPU servers.
+// See https://www.nginx.com/blog/socket-sharding-nginx-release-1-9-1/ for more details :)
+//
+// The package is based on https://github.com/kavu/go_reuseport .
```

Comment:
Please remove this whole comment. It should only be present in one file.

Reply:
Done